freshmaker/pyxis.py

```diff
@@ -314,3 +315,50 @@ def is_bundle(self, nvr):
             if label["name"] == "com.redhat.delivery.operator.bundle" and label["value"] == "true":
                 return True
         return False
+
+    def get_image_metadata_by_nvr(self, nvr):
```

Comment:
`get_image_metadata_by_nvr` and `get_rebuild_image_id` are ambiguous names or not describing the purpose very well, I'd suggest something like this (not tested):
```
def image_is_tagged_auto_rebuild(self, nvr):
    images = get_images_by_nvr(nvr, include=...)
    for image in images:
        for repo in image['repositories']:
            if not repo['published']:
                continue
            auto_rebuild_tags = self.get_auto_rebuild_tags(repo['registry'], repo['repository'])
            if set(image['tags']) & set(auto_rebuild_tags):
                return True
    return False
```

Reply:
Ok. Create a new method get_image_metadata_by_nvr is considering to use it when getting rpm manifest data in future. But yeah, let's consider it in following patch, maybe we don't need it. Ack. Fixing